setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+
+setup(
+    name='mittn',
+    use_scm_version=True,
+    description='Mittn',
+    long_description='',
```

Comment:
You could re-use something from the README for the long_description, e.g. "Mittn is an evolving suite of security testing tools to be run in Continuous Integration context."